### PR TITLE
[bitnami/geode] Add tests and publishing using VIB

### PR DESCRIPTION
**Pull request by FraPazGal**
### Description of the change

The main objective of this PR is to publish our Bitnami Geode Helm Chart using VMware Image Builder. In order to do that, several changes are included:

- Adding the asset to the publishing workflow
- Increasing the existing test coverage of the asset by adding Cypress tests.
- Increasing the existing test coverage of the asset by adding GOSS tests.

### Benefits

- Ensuring higher quality of the Helm charts.
- Increased pool of assets completely handled by VMware Image Builder.

### Possible drawbacks

Automated tests could show to be potentially flaky. 

### Applicable issues

NA

### Additional information

Tested in https://github.com/FraPazGal/charts/pull/112

### Checklist

- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: .vib/geode/goss/goss.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    owner: root
+  /opt/bitnami/geode/config/gemfire.properties:
+    exists: true
+    filetype: file
+    mode: "0644"
+    group: root
+    contains:
+    - http-service-port={{ .Vars.locator.containerPorts.http }}
+    - jmx-manager-port={{ .Vars.locator.containerPorts.rmi }}
+command:
```
Comment (joancafom) [inline .vib/geode/goss/goss.yaml:15]: I noticed that we are using a custom `sysctls` override setting in the `runtime_params` to disable TCP SYN cookies. According to the [official documentation](https://geode.apache.org/docs/guide/114/managing/monitor_tune/disabling_tcp_syn_cookies.html),  this is important to apply or else performance would be impacted.

I believe we should add a test to check the status of this setting, something like:

```
$ sysctl -a | grep "net.ipv4.tcp_syncookies"
```

File: .vib/geode/vib-publish.json [hunk 2]
```diff
@@ -22,11 +22,11 @@
...
...ogICAgZW5hYmxlZDogdHJ1ZQogICAgZnNHcm91cDogMTAwMgogICAgc3lzY3RsczoKICAgICAgLSBuYW1lOiBuZXQuaXB2NC50Y3Bfc3luY29va2llcwogICAgICAgIHZhbHVlOiAiMCIKICBjb250YWluZXJTZWN1cml0eUNvbnRleHQ6CiAgICBlbmFibGVkOiB0cnVlCiAgICBydW5Bc1VzZXI6IDEwMDIKICBzZXJ2aWNlOgogICAgcG9ydHM6CiAgICAgIGxvY2F0b3I6IDEwMzM1CiAgICAgIGh0dHA6IDgwCiAgICAgIHJtaTogMTEwMAogICAgc2Vzc2lvbkFmZmluaXR5OiBDbGllbnRJUAogICAgdHlwZTogTG9hZEJhbGFuY2VyCnNlcnZlcjoKICByZXBsaWNhQ291bnQ6IDMKICBjb250YWluZXJQb3J0czoKICAgIHNlcnZlcjogNDA0MDMKICAgIGh0dHA6IDcwNjkKICAgIHJtaTogMTA5OAogICAgbWV0cmljczogOTkxNAogIHJlYWRpbmVzc1Byb2JlOgogICAgZW5hYmxlZDogdHJ1ZQogICAgaW5pdGlhbERlbGF5U2Vjb25kczogMTIwCiAgICB0aW1lb3V0U2Vjb25kczogMzAKICAgIGZhaWx1cmVUaHJlc2hvbGQ6IDUKICBsaXZlbmVzc1Byb2JlOgogICAgZW5hYmxlZDogdHJ1ZQogICAgaW5pdGlhbERlbGF5U2Vjb25kczogMTIwCiAgICB0aW1lb3V0U2Vjb25kczogMzAKICAgIGZhaWx1cmVUaHJlc2hvbGQ6IDUKICBwb2RTZWN1cml0eUNvbnRleHQ6CiAgICBlbmFibGVkOiB0cnVlCiAgICBmc0dyb3VwOiAxMDAyCiAgICBzeXNjdGxzOgogICAgICAtIG5hbWU6IG5ldC5pcHY0LnRjcF9zeW5jb29raWVzCiAgICAgICAgdmFsdWU6ICIwIgogIGNvbnRhaW5lclNlY3VyaXR5Q29udGV4dDoKICAgIGVuYWJsZWQ6IHRydWUKICAgIHJ1bkFzVXNlcjogMTAwMgogIHNlcnZpY2U6CiAgICBwb3J0czoKICAgICAgc2VydmVyOiA0MDQwMwogICAgICBodHRwOiA4MAogICAgICBybWk6IDEwOTg=",
```
Comment (joancafom) [inline .vib/geode/vib-publish.json:25]: You are missing `auth.enabled=true`.

**Review by joancafom — COMMENTED** [review]
Thanks for your PR @FraPazGal ! I left you a couple of comments for you to review when you have the chance.

File: .vib/geode/goss/goss.yaml (hunk 1 above)
Comment (FraPazGal) [reply .vib/geode/goss/goss.yaml:15]: Good idea, added!

**Review by agomezmoron — APPROVED** [review]

**Review by joancafom — APPROVED** [review]

**Pull request merged**